### PR TITLE
Moves get_mean_condition_scores sql call to a separate utils file

### PR DESCRIPTION
**Pull request by strangeties**
1) Moves mean condition scores sql call to a separate utils file.
2) Adds logic for pulling mean condition scores from db (rather than computing every time).

### CODE REVIEW

**Comment by strangeties** [comment]
working through how data would be saved/pulled from db: this warrants a few more commits - converting to draft for now

**Comment by strangeties** [comment]
update: ready for review (assuming checks pass)

File: src/planscape/plan/views.py [hunk 1]
```diff
@@ -1,28 +1,25 @@
+from planscape import settings
```
Comment (riecke) [inline src/planscape/plan/views.py:1]: Nit: please run "Sort Imports"; the datetime and json imports should come first, I think.

**Review by riecke — APPROVED** [review]
Nice---and good tests.

One more nit: please file a bug to recompute the averages if the planning area changes.  Maybe we'll never let the user change the planning area? but if we do, there will be a
bug from the assumptions made here.

File: src/planscape/plan/views.py (hunk 1 above)
Comment (strangeties) [reply src/planscape/plan/views.py]: ok - done.
I've also filed #431 .

**Comment by strangeties** [comment]
Thanks for reviewing.

**Comment by strangeties** [comment]
#394

**Pull request merged**